controllers/nvidiadriver_controller.go

```diff
@@ -168,6 +169,31 @@ func (r *NVIDIADriverReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return reconcile.Result{}, nil
 	}
 
+	operatorNamespace := os.Getenv("OPERATOR_NAMESPACE")
```

Comment:
As an observation, I see we already fetch this env in `main.go` https://github.com/NVIDIA/gpu-operator/blob/d71f30d0d2a1009b52ad5dd41ca8a1a878b7f11e/cmd/gpu-operator/main.go#L109 Thoughts on adding the operator namespace as a field of the `NVIDIADriverReconciler` so that we don't have to repeatedly source this env during every reconciliation?

Reply:
Yes, I had the same thought as well. I was thinking of making this change in both the cluster policy and nvidia driver reconcilers, so we call the os.GetEnv only once. I was thinking of doing this in a follow up PR

Reply:
These are three other places where we do this

- https://github.com/NVIDIA/gpu-operator/blob/d68374e891d6a14242dfd9e48eac88b34c80420d/internal/state/driver.go#L621
- https://github.com/NVIDIA/gpu-operator/blob/f9e0eb7b425813a4800da8401318049e6f90b1da/cmd/gpu-operator/main.go#L109
- https://github.com/NVIDIA/gpu-operator/blob/2e5685a9481c82df67ad07dfb506b2b1d0b9fd1e/controllers/state_manager.go#L762